mars-sim-core/src/main/java/com/mars_sim/core/structure/Settlement.java

```diff
@@ -917,54 +914,6 @@ public boolean timePassing(ClockPulse pulse) {
 		return true;
 	}
 	
-	/**
-	 * Checks and reports on the average mineral site improvement made.
-	 * 
-	 * @param sol
-	 */
-	private void reportsMineralSiteImprovement(int sol) {
```

Comment:
Can we still have this method for reporting mineral site improvement at the end of a sol ?

Reply:
It is an expensive method and the output just goes to the console. If you want to record the historical improvements of sites then lets do it probably and uses our Metrics Manager idea